templates/core/crowdfunding_donors.html

```diff
@@ -0,0 +1,50 @@
+{% extends 'global/base.html' %}
+{% load staticfiles rev %}
+{% load element_by_counter %}
+
+{% block extra_css %}
+    <link rel="stylesheet" type="text/css" href="{% static_rev 'css/donors.css' %}">
+{% endblock %}
+
+{% block content %}
+    <div class="container">
+        <div class="row">
+            <h2>💖 Special thanks to all our amazing supporters! 💖</h2>
+            <br />
+            <div class="col-md-6">
+                <p>
+                This year (2018) we ran a crowdfunding campaign on Indiegogo's generosity website from
+                the 29th January to the 29th of March with the aim of raising US $10,000. We are truly
+                grateful to all our awesome supporters who helped reach our target goal. We managed to
+                raise a total of $13,314 - 133% of our goal!
+                </p>
+            </div>
+            <div class="col-md-6">
+                <img src="{% static 'img/global/donate/django_crowdfunding_heart.png' %}"></a>
+            </div>
+        </div>
+
+        <br/>
+
+        <div class="row">
+            <h2>Our awesome supporters ✨</h2>
+            {% for donor in donor_list %}
+                 {% if forloop.first %}<div class="row">{% endif %}
+                <div class="col-md-4">
+                    <h4 class="donor-text">{{ donor.name }}</h4>
+                </div>
+                {% if forloop.counter|divisibleby:3 %}</div><div class="row">{% endif %}
+                {% if forloop.last %}</div>{% endif %}
+                {% if forloop.counter|divisibleby:42 %}
+                    <br />
+                    {% element_by_counter quotes forloop.counter 42 as quote %}
```

Comment:
`forloop.counter` starts from `1`, not `0` - that means that first quote will never be displayed. Instead we should use `forloop.counter0` (but only in this line!)